src/helpers/ci.ts

```diff
@@ -546,3 +546,40 @@ const parsePipelineNumber = (pipelineNumberStr: string | undefined): number | un
     return isFinite(pipelineNumber) ? pipelineNumber : undefined
   }
 }
+
+export const getCIEnv = (): {ciEnv: Record<string, string>; provider: string} => {
```

Comment:
why do you need a new util to extract the environment? Couldn't you reuse the ones we use for `junit`?

Reply:
The junit one remaps things to the tags we use so things like `ci.pipeline.name`. For this to work we need the runner env variables including some that we do not map to any of our specific ci tags.